blog/_posts/2025-04-22-vm-generations-and-aks.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "VM Generations and AKS"
+description: "Learn more about Generation 1 and Generation 2 VMs and what they offer, differences between them, upcoming Generation 1 VM retirements, and how to move your AKS workloads to Generation 2 VMs."
+date: 2025-04-22 # date is important. future dates will not be published
+authors: 
+   - Jack Jiang
+categories: 
+- operations
+---
+*This blog was co-authored by Ally Ford, Product Manager 2, AKS, Sarah Zhou, Product Manager, Azure Compute, and Jack Jiang, Product Manager, AKS*
+## What are Virtual Machine Generations?
+
+Azure virtual machine sizes are broken down by the amount/type of resources allocated to the virtual machine in the cloud. These resources are in turn 
+tied to a portion of a physical server's hardware capabilities. A single physical server may be broken down into a collection of *size series*, or 
+configurations available utilizing its resources. 
+
+As the physical hardware ages and newer components become available, older hardware of a previous generation get retired, while newer generation hardware 
+is progressively made available. 
+
+In this blog, we will go over Generation 1 and the newer Generation 2 virtual machines. Both have their own use cases, and [picking the right one](https://learn.microsoft.com/windows-server/virtualization/hyper-v/plan/should-i-create-a-generation-1-or-2-virtual-machine-in-hyper-v) 
+to suit your workloads is critical in ensuring you get the best possible experience, capabilities, and cost.  
+
+### Virtual Machine Generation Overview
+
+Azure VM sizes (v5 and older) have largely supported both Generation 1 and Generation 2 VMs. [This page](https://learn.microsoft.com/azure/virtual-machines/generation-2)
+gives a thorough breakdown on VM series and the generation they support. 
+
+The latest v6 VMs (whether they are 
+[Intel](https://techcommunity.microsoft.com/blog/azurecompute/announcing-preview-of-new-azure-dlsv6-dsv6-esv6-vms-with-new-cpu-azure-boost-and/4192124), 
+[AMD](https://techcommunity.microsoft.com/blog/azurecompute/new-daeafav6-vms-with-increased-performance-and-azure-boost-are-now-generally-av/4309381), or 
+[ARM](https://azure.microsoft.com/blog/azure-cobalt-100-based-virtual-machines-are-now-generally-available/?msockid=150f515112e461e5201d45b1136e602c)
+will exclusively support Generation 2 VMs.
+
+### Comparing Generation 1 & Generation 2
+Generation 2 VMs offer exclusive features over Generation 1 VMs, such as increased memory, improved CPU performance, support for NVMe disks, and support for Trusted 
+Launch. With some [exceptions](https://learn.microsoft.com/windows-server/virtualization/hyper-v/plan/should-i-create-a-generation-1-or-2-virtual-machine-in-hyper-v),
+it is generally recommended to migrate to Generation 2 VMs to take advantage of the newest features and functionalities in Azure VMs.
```

Comment:
do we need a doc on how to do this on AKS? 

The user doesn't really have control over this but they don't really need it to migrate they just need

a) to know they are in gen 1 (we need to show them how to check)
b) to know how to migrate, eg. deploy a nodepool with a Gen2 supported SKU, and we should use Gen2 on those cases (I think windows was the last exception, right @allyford ?)